CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
+
+project(sundown-apiary VERSION 1.0 LANGUAGES C)
+
+add_library(sundown-apiary
+        src/autolink.c
```

Comment:
I noticed here you are using 8-spaces indentation, but in install and target_include_directories 4 spaces. Is that a mistake or is this intentional/normal in cmake?

Reply:
Yeah, that's a mistake.